### PR TITLE
Cayden reviewer

### PR DESCRIPTION
**Pull request by ccodel**
Reviewer 3 claims that "the authors should at least say why regular symmetry breaking clauses cannot be used" and that "the intuition behind Lex order needs to be explained." This PR addresses these reviewer concerns in Section 5 (symmetry breaking).

Notably, see the final sentence of paragraph 1, all of paragraph 2, and the two paragraphs after the \end{definition}.

### CODE REVIEW

File: ITP/symmetry-breaking.tex [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
+such as structured bounded variable addition~\cite{sbva},
+but it is accepted wisdom in the SAT-solving community that problem-specific symmetry breaking is more effective.
 
-In the context of the Empty Hexagon Number, the symmetry breaking done by Heule and Scheucher consists in assuming
-that in order to search for a list of $30$ points in general position without a $6$-hole,
-it suffices to can search only amongst lists of $30$ points in \emph{canonical} position.
-These are defined as follows.
+The symmetry breaking performed by Heule and Scheucher for the Empty Hexagon Number places the points in \emph{canonical position},
+which enforces several notions of order.
+By proving that for any list of points in general position,
```
Comment (JamesGallicchio) [inline ITP/symmetry-breaking.tex]: don't love how this sentence reads. maybe instead something like:
"For any list of points in general position, there exists a list of points in canonical position with the same orientations. (something about how this is an orientation property)"

File: ITP/symmetry-breaking.tex [hunk 2]
```diff
@@ -1,32 +1,50 @@
...
+there exists a list of points in canonical position with the same orientations,
+we can work with the latter when encoding the problem into SAT.
+Canonical position is defined as follows.
```
Comment (JamesGallicchio) [inline ITP/symmetry-breaking.tex:12]: this sentence might be unnecessary honestly

File: ITP/symmetry-breaking.tex [hunk 3]
```diff
@@ -1,32 +1,50 @@
...
-preserves the presence of $k$-holes, or convex $k$-gons.
-This is the reason for incorporating the \emph{parity} flag in the definition of $\sigma$-equivalence.
-\input{fig-sigma-equiv.tex}
-As illustrated in~\Cref{fig:sigma-equiv}, there are pointsets that are only $\sigma$-equivalent to their reflections
-with \lstinline|parity := true|.
-We are now ready to state the main symmetry breaking theorem and sketch its proof.
+The three ordering properties each break a different symmetry.
+First, the $x$-order property breaks symmetry due to how we label the points by ensuring that the points are labeled from left to right.
+Plus,
+the $x$-order does more than break symmetry:
```
Comment (JamesGallicchio) [inline ITP/symmetry-breaking.tex]: this seems potentially confusing to the reader -- maybe say "this assumption also substantially simplifies the encoding of clauses~\labelcref{...}"

File: ITP/symmetry-breaking.tex (hunk 1 above)
Comment (ccodel) [reply ITP/symmetry-breaking.tex]: Fair enough. I incorporated your suggestion.

File: ITP/symmetry-breaking.tex (hunk 2 above)
Comment (ccodel) [reply ITP/symmetry-breaking.tex:12]: Gone!

File: ITP/symmetry-breaking.tex (hunk 3 above)
Comment (ccodel) [reply ITP/symmetry-breaking.tex]: Incorporated.

**Comment by ccodel** [comment]
James' comments have been addressed. Ready for another round of review!

File: ITP/symmetry-breaking.tex (hunk 2 above)
Comment (JamesGallicchio) [reply ITP/symmetry-breaking.tex:12]: (I think it is not actually gone in latest revision?)

Comment (ccodel) [reply ITP/symmetry-breaking.tex:12]: I'm confused. When I click [this commit](https://github.com/bsubercaseaux/EmptyHexagonLean/pull/3/commits/1f9265edbda1a0da4acf87a5f1b06a9302f69d49), I see red on the previous line and green on my new edits.

Did the force push break things? :(

**Comment by bsubercaseaux** [comment]
Looks good.

**Pull request merged**